_posts/en/newsletters/2018-12-11-newsletter.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Bitcoin Optech Newsletter #25"
+permalink: /en/newsletters/2018/12/11/
+name: 2018-12-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter suggests helping test a Bitcoin Core maintenance
+release candidate, provides a link to a modern block explorer whose code
+has been open sourced, and briefly describes a suggestion for signature
+hashes to optionally cover transaction size.  Notable code changes made
+in the past week to popular infrastructure projects are also described.
+
+## Action items
+
+- **Help test Bitcoin Core 0.17.1RC1:** the first release candidate for
+  this [maintenance release][] has been tagged.  Testing by businesses
```

Comment:
Include link to tagged release? https://github.com/bitcoin/bitcoin/releases/tag/v0.17.1rc1